CHANGELOG.md

```diff
@@ -9,6 +9,7 @@
 * Resolved the partially styled user interface issue that occurs when the style theme is switched between the `NightStyle` and `DayStyle`, after the resume  button was tapped. ([#1589](https://github.com/mapbox/mapbox-navigation-ios/pull/1589))
 * Fixed a rare crash associated with the keyboard when showing the end of route view controller. [#1599](https://github.com/mapbox/mapbox-navigation-ios/pull/1599/)
 * Added a `MapboxVoiceController.audioPlayer` property. You can use this property to interrupt a spoken instruction or adjust the volume. [#1596](https://github.com/mapbox/mapbox-navigation-ios/pull/1596)
+* Exposed `automaticallyAdjustsStyleForTimeOfDay`, `delegate`, and `styles` on `StyleManager` publicly. [#1617](https://github.com/mapbox/mapbox-navigation-ios/pull/1617)
```

Comment:
From the developer’s point of view, these properties never existed in the first place. Also, qualify each property reference so jazzy can automatically link to the documentation when the changelog appears in the docset cover:

> Added `StyleManager.automaticallyAdjustsStyleForTimeOfDay`, `StyleManager.delegate`, and `StyleManager.styles` properties so that you can control same time-based style switching just as `NavigationViewController` does.